pkg/systemd/services/service-details.jsx

```diff
@@ -438,6 +492,8 @@ export class ServiceDetails extends React.Component {
         const tooltipMessage = enabled ? _("Stop and disable") : _("Start and enable");
         const hasLoadError = this.props.unit.LoadState !== "loaded" && this.props.unit.LoadState !== "masked";
         const loadError = this.props.unit.LoadError ? this.props.unit.LoadError[1] : null;
+        console.log(hasLoadError, loadError);
+        console.log(this.state.error);
```

Comment:
Please remember to clean these up once the PR works.

Reply:
Yes, sorry :(